packages/react-charting/docs/TestingStrategy.md

```diff
@@ -0,0 +1,119 @@
+**Details**
+
+<img width="455" alt="image" src="https://github.com/srmukher/fluentui/assets/120183316/261b1944-b0f3-4e4f-80a3-9f8f88d9f82f">
```

Comment:
the image src is pointing to your local user repo

Reply:
added as images to images folder.